{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -221,11 +221,11 @@ def env(name, default=NoDefaultValue, type_=str):
     type_=(lambda v: tuple(v.split(":", 1)) if (v is not None and ":" in v) else None),
 )
 SECURE_SSL_REDIRECT = env("SECURE_SSL_REDIRECT", default=True, type_=boolish)
-SESSION_COOKIE_SECURE = env("SESSION_COOKIE_SECURE", default=False, type_=boolish)
-CSRF_COOKIE_SECURE = env("CSRF_COOKIE_SECURE", default=False, type_=boolish)
-SECURE_HSTS_SECONDS = env("SECURE_HSTS_SECONDS", default=0, type_=int)
+SESSION_COOKIE_SECURE = env("SESSION_COOKIE_SECURE", default=SECURE_SSL_REDIRECT, type_=boolish)
+CSRF_COOKIE_SECURE = env("CSRF_COOKIE_SECURE", default=SECURE_SSL_REDIRECT, type_=boolish)
+SECURE_HSTS_SECONDS = env("SECURE_HSTS_SECONDS", default=3600 if SECURE_SSL_REDIRECT else 0, type_=int)
 SECURE_HSTS_INCLUDE_SUBDOMAINS = env(
-    "SECURE_HSTS_INCLUDE_SUBDOMAINS", default=False, type_=boolish
+    "SECURE_HSTS_INCLUDE_SUBDOMAINS", default=True, type_=boolish
```

Comment:
My only worry is running into HSTS issues with `localhost`; is that something you feel OK with? If so, 👍 .

Reply:
@wlonk hmm, let me revisit that tomorrow

Reply:
@wlonk I added a line to turn HSTS off in the local settings. Not foolproof if you use a different settings file locally, but should cover most cases.